lib/capybara/selenium/logger_suppressor.rb

```diff
@@ -3,7 +3,7 @@
 module Capybara
   module Selenium
     module DeprecationSuppressor
-      def initialize(*, **)
+      def initialize(...)
```

Comment:
I was about to point out that this is incompatible with Ruby 2.5, but then I took a closer look at the gemspec and saw the `required_ruby_version` :stuck_out_tongue: . I'm sorry, I should have done it this way from the beginning.